FASTN.ftd

```diff
@@ -394,6 +396,13 @@ favicon: /-/fastn.com/images/favicon.svg
 - Dynamic Redirect: /ftd/redirect/
   document: backend/ftd-redirect.ftd
 
+## Learn: /backend/learn/
+  document: /backend/dynamic-ui/index.ftd
+
+- Creating Dynamic UI: /country-details/
+  document: /backend/dynamic-ui/index.ftd
```

Comment:
This file is all wrong. Why is URL country detail but file name dynamic-ui? Everything in backend section is dynamic ui. 

Rename it to backend/examples/country-details.ftd.

Reply:
Updated it like this:
```ftd
- Country Details: /country-details/
  document: /backend/country-details/index.ftd
  - Basic of `http`, `data modelling`: /country-details/basics/
    document: /backend/country-details/http-data-modelling.ftd